### ISSUE TITLE
calculate accuracy on validation set

### ISSUE THREAD

**Issue by philippmuench** [hiddengenome/deepG#42]

**Comment by philippmuench** [comment]
seem it only works with a custom callback

see "Callback for logging metrics" here http://thecooldata.com/2018/12/changing-learning-rate-during-training-on-each-batch-iteration-using-callbacks-in-r-keras/
